myhoard/backup_stream.py

```diff
@@ -255,6 +255,15 @@ def binlog_upload_age(self):
             # If we're not supposed to be streaming binlogs then report age as 0
             return 0
 
+    @property
+    def binlog_upload_delay(self):
+        """Returns the number of seconds since the oldest pending binlog was written."""
+        mtimes = {os.stat(binlog["full_name"]).st_mtime for binlog in self.state["pending_binlogs"]}
```

Comment:
I think this can lead to a race condition if the binlog is removed from disk after being read from state but before being statED. I cannot check which thread is doing what right now, but his should be taken into account and the exception on stat ignored.

Reply:
ok, done.